frontend/lib/ui/style/form.less

```diff
@@ -1,7 +1,7 @@
 form {
   input {
     border: 1px solid var(--bg-gray-darkest);
-    background-color: #fff;
+    background-color: var(--base-white);
     font-size: 16px;
```

Comment:
Not part of your change, but do we want font-sizes to be defined in px here? Shouldn't this be `1rem`? (applies to other font-size definitions as well)

Reply:
Replaced most px for rem in the whole codebase in 3daa4d9e0b473944f6053ce90993cd290237f796